test/e2e/httpproxy/httpproxy_test.go

```diff
@@ -193,93 +193,109 @@ var _ = Describe("HTTPProxy", func() {
 	})
 
 	f.NamespacedTest("httpproxy-backend-tls", func(namespace string) {
```

Comment:
hm, we might need to separate out the tests into different `NamespacedTest` blocks to make sure no test pollution

if we see flakiness in this area (creating/deleting resources etc.) this would be something to look into